common/src/androidTest/java/com/microsoft/identity/common/internal/platform/DevicePoPManagerEncryptionTests.java

```diff
@@ -80,7 +80,7 @@
 
     public DevicePoPManagerEncryptionTests(final IDevicePopManager.Cipher cipher)
```

Comment:
nit: also rename class name?

Reply:
Same for all classes below.

- If we can migrate them to 4j, let's do that. 
- If not, 
     - if these tests are speficially for android, rename the class to AndroidXXXXXX 
     - Otherwise, left a TODO.

Reply:
I've added a todo to investigate their migration to common4j